alws/crud/products.py

```diff
@@ -332,13 +400,15 @@ async def modify_product(
                 raise ProductError(error_msg)
         if modification == 'remove' and not force:
             if db_build not in db_product.builds:
-                error_msg = f'Packages of build {build_id} cannot be removed ' \
-                            f'from {product} product ' \
+                error_msg = f'Packages of build {build_id} cannot ' \
+                            f'be from {product} product ' \
                             f'as they are not added there'
                 raise ProductError(error_msg)
 
     pulp_client = PulpClient(settings.pulp_host, settings.pulp_user,
                              settings.pulp_password)
+    await set_platform_for_products_repos(db=db, product=db_product)
+    await set_platform_for_build_repos(db=db, build=db_build)
```

Comment:
Why we're invoke commit in these functions? I believe that it's unnecessary, because we can add them in session later or even do not add them (but I'm not sure would it work or not)
i.e.:
```python3
await set_platform_for_products_repos(db=db, product=db_product)
await set_platform_for_build_repos(db=db, build=db_build)
...
db.add_all([db_product, db_build])
await db.commit()
```